modules/client-configuration/pages/registration-methods-saltbundle.adoc

```diff
@@ -0,0 +1,33 @@
+[[registration-methods-saltbundle]]
+= Client Registration with Salt Bundle
+
+The registration method with the Salt bundle is the recommended registration method as long as it does not cover your use case.
+This section explains the advantages and limitations of the current implementation.
+The Salt bundle is provided as the [package]``venv-salt-minion`` that consists of Salt, Python, and the Python modules Salt depends on.
+Using the Salt bundle it is no longer needed that the client provides the required Python version or any Python modules.
```

Comment:
Bootstrapping with web UI still requires Python to be installed on the client. Will be changed on implementing Salt-SSH with Salt Bundle

Reply:
locally changed accordingly.  will push later.